Sources/NIOSSL/SSLContext.swift

```diff
@@ -520,6 +537,78 @@ extension NIOSSLContext {
     }
 }
 
+extension NIOSSLContext {
+    /// A collection of buffers representing a STACK_OF(X509_NAME)
+    struct NIOX509NameListBuffers {
```

Comment:
I think if we're only going to use `count` then for now let's save ourself some complexity and just have `getX509NameListCount` call `CNIOBoringSSL_sk_X509_NAME_num` directly. Otherwise, we should arrange a more complex implementation that can actually be iterated, but for now I think that's excessive.

Reply:
Great idea!

Reply:
Resolved in the latest commit.